testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -760,6 +719,47 @@ impl ClusterSwarm for ClusterSwarmKube {
             workspace
         ))
     }
+
+    async fn put_file(
+        &self,
+        node: &str,
+        pod_name: &str,
+        path: &str,
+        content: Vec<u8>,
+    ) -> Result<()> {
+        let bucket = "toro-cluster-test-flamegraphs";
```

Comment:
I found this bucket name has been used somewhere else, I think u may can create a followup task/PR to make this point into config or global scope, which can be shared and easy access

Reply:
Yes, will do. We also need to change the name.